### PR TITLE
Skeleton loader match to respective layout

### LINKED ISSUE #415: Skeleton reconstruction
# Description

reconstructing skeleton loaders to match respective page layouts

### PR DESCRIPTION
**Pull request by Dawaic6**
# PR Description

This PR reconstructs the skeleton loaders across the entire application to better match the respective page layouts. The goal is to enhance the user experience by providing more accurate visual placeholders that align with the design and content of each page while the data is loading.

# Description of tasks that were expected to be completed

-  Redesigned skeleton loaders for the **ttl page** to reflect the table and filter input layout.
-  Refined skeleton loaders for the **Admins page** to match the admin list and control components.
-  Implemented loaders for the **Domains page** to mirror the grid and form layout for domain management.
-  Created loaders that fit the **Calendar page's** calendar and event list layout.
-  Ensured consistency in loader colors, sizes, and spacing across all pages.
-  Made all skeleton loaders responsive to adapt well to different screen sizes.

# How has this been tested?

**Manual Testing:** Verified each page's skeleton loaders to ensure they align with the respective page layout and design on various devices (desktop, tablet, mobile).

- Navigated through each page to check that loaders appear correctly while data is being fetched.
- Checked that the loaders are responsive and adapt well to different screen sizes.
- Ensured no existing functionality was broken by the new loaders.

# Screenshots (If appropriate)
- **calender**
image
- **organisation**
image
- ttl page

image

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] My test coverage meet the set test coverage threshold
- [x] There are no vulnerabilities
- [x] There are no conflicts with the base branch

### CODE REVIEW

**Review by andremugabo — APPROVED** [review]
LGTM

**Review by janet-barbie — APPROVED** [review]

**Review by shema-surge — APPROVED** [review]
Can u put some space between the "add Event" button and the skeleton on the calendar page

**Review by JacquelineTuyisenge — APPROVED** [review]

File: src/Skeletons/calSkeleton.tsx
Comment (ncutixavier) [inline src/Skeletons/calSkeleton.tsx:1]: We need to have meaningful files (table.skeleton.tsx)

File: src/components/Calendar.tsx [hunk 1]
```diff
@@ -46,44 +47,41 @@ const Calendar = () => {
         }));
         setData(all);
       } catch (error) {
-        console.log({ eventsError: data });
+        console.log({ eventsError: error });
```
Comment (ncutixavier) [inline src/components/Calendar.tsx]: Remove all unnecessary logs

File: src/components/Calendar.tsx [hunk 2]
```diff
@@ -46,44 +47,41 @@ const Calendar = () => {
...
+    >
+      <p className="px-3 py-1 ">{e.event.title}</p>
+      <p className="px-3 py-1 ">{e.event.extendedProps.hostName}</p>
+      <p className="px-3 py-1 ">
+        {e.event.extendedProps.timeToStart} -{' '}
+        {e.event.extendedProps.timeToFinish}
+      </p>
+      <ReactTooltip data-html={true} />
+    </div>
+  );
```
Comment (ncutixavier) [inline src/components/Calendar.tsx]: Avoid unnecessary changes anywhere

**Review by ncutixavier — CHANGES_REQUESTED** [review]
In general, Unnecessary changes/formats are not allowed

**Review by ncutixavier — APPROVED** [review]

**Review by Bananayosostene — APPROVED** [review]

**Review by Salim-54 — COMMENTED** [review]
Resolve the conflicts

**Pull request merged**